tests/python/test_fields_builder.py

```diff
@@ -6,171 +6,175 @@
 
 @ti.test(arch=[ti.cpu, ti.cuda, ti.vulkan, ti.metal])
 def test_fields_with_shape():
-    n = 5
-    x = ti.field(ti.f32, [n])
+    shape = 5
+    x = ti.field(ti.f32, shape=shape)
+    y = ti.field(ti.f32, shape=shape)
 
     @ti.kernel
-    def func():
-        for i in range(n):
+    def assign_field_single():
+        for i in range(shape):
             x[i] = i
 
-    func()
-
-    for i in range(n):
+    assign_field_single()
+    for i in range(shape):
         assert x[i] == i
 
-    y = ti.field(ti.f32, [n])
```

Comment:
This line shouldn't be moved to the top - it implicitly uses the field builder.

Reply:
However, these two variables `x` and `y` are used by multi-functions.

Reply:
`assign_field_single` doesn't use `y` so it's fine to leave `y` here right?

Reply:
Oh, I know your means.

Reply:
Fixed.